06-single-hop-udp/06-single-hop-udp.md

```diff
@@ -70,3 +70,38 @@ Sending UDP from one iotlab-m3 node to a non-existent neighbor.
 
 No leaks in the packet buffer (check with `gnrc_pktbuf_cmd`).
 Packet loss is irrelevant for this test.
+
+Task #05 - Empty UDP on native
+==============================
+### Description
+
+Sending UDP between two native nodes.
+* Stack configuration:    IPv6 (default)
+* Count:                  1000
```

Comment:
I suggest to lower the values to 10pkts, every 100ms, here and below.
Noone wants to wait for a 1000 0b packets to not crash the test system after the first 5 or so have succeeded.

Reply:
and then maybe loss <= 10%, so one packet can get lost for whatever reason without failing the test.

Reply:
Done.